var/spack/repos/builtin/packages/miktex/package.py

```diff
@@ -0,0 +1,47 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Miktex(CMakePackage):
+    """MiKTeX is a modern C/C++ implementation of TeX & Friends for Windows,
+    macOS and Linux."""
+    maintainers = ['vmiheer']
+    homepage = "https://miktex.org/"
+    url = "https://miktex.org/download/ctan/systems/win32/miktex/source/miktex-22.1.tar.xz"
```

Comment:
Why not use the GitHub version (`https://github.com/MiKTeX/miktex/archive/refs/tags/22.1.tar.gz`)?

The sha256 is different (`a9013a424a5a70ddab08c6b8f1693a22d5da9081399228c37a80a2e733642df2`).